back-end/routes/routes.js

```diff
@@ -22,4 +23,5 @@ module.exports = server => {
     server.route('/browse').get(fetchImages);
     server.route('/collectionAdd').post(collection);
     server.route('/currentuser/').get(currentuser);
+    server.route('/addImageToCollection').post(collection);
   };

```

Comment:
shouldn't this be going to the addImageToCollection controller, not collection.

also, it's not in this PR, but related.  in the addImageToColleciton controller is this line:

```
           user.credits -= Number(credits)
```
I don't see where the credits variable is coming from.  Adding an image to a collection costs one credit right?  So shouldn't you be just subtracting one credit here?